src/Optimizer/Transformer/GoogleFontsPreconnect.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace AmpProject\Optimizer\Transformer;
+
+use AmpProject\Dom\Document;
+use AmpProject\Optimizer\ErrorCollection;
+use AmpProject\Optimizer\Transformer;
+
+/**
+ * Transformer adding resource hints to preconnect to the Google Fonts domain.
+ *
+ * @package ampproject/amp-toolbox
+ */
+final class GoogleFontsPreconnect implements Transformer
+{
+
+    /**
+     * Domain that the Google Fonts static files are loaded from.
+     *
+     * @var string
+     */
+    const GOOGLE_FONTS_STATIC_DOMAIN = 'https://fonts.gstatic.com';
```

Comment:
Since to me `domain` would be just the host like `fonts.gstatic.com`

```suggestion
    const GOOGLE_FONTS_STATIC_ORIGIN = 'https://fonts.gstatic.com';
```